web/sites/default/config/core.entity_view_display.node.event.default.yml

```diff
@@ -186,14 +165,6 @@ content:
             classes: {  }
     type: entity_reference_entity_view
     region: content
-  field_neighborhood:
```

Comment:
@brittanysmart, did you remove neighborhood and topics on purpose?

Reply:
No, that's a mistake. There should only be a template change.

Reply:
I'm going to fail it then. This is the only file in the PR. :)